assets/js/gamesData.json

```diff
@@ -2099,9 +2099,37 @@
   "thumbnailUrl": "Screen_Pet_Game.png"
 
 },"416":{
+
+
+  "gameTitle": "rapid_click_frenzy",
+  "gameUrl": "rapid_click_frenzy",
+  "thumbnailUrl": "rapidgame.png"
+
+ 
+  
+}
+  ,"417":{
```

Comment:
remove all these entry and place your below each entry with no 417
or arrange the entry in the order 
417
409
418
410
and so on at the bottom of file